lib/WeBWorK/DB.pm

```diff
@@ -97,14 +97,12 @@ use warnings;
 use Carp;
 use Data::Dumper;
 use Scalar::Util qw/blessed/;
-use HTML::Entities qw( encode_entities );
-use Mojo::JSON qw(encode_json decode_json);
-
 use WeBWorK::DB::Schema;
 use WeBWorK::DB::Utils qw/make_vsetID grok_vsetID grok_setID_from_vsetID_sql
 	grok_versionID_from_vsetID_sql/;
 use WeBWorK::Debug;
 use WeBWorK::Utils qw(runtime_use);
+use HTML::Entities qw( encode_entities );
```

Comment:
I had moved this up because `HTML::Entities` is not a WeBWorK package, but from CPAN. Isn't that how we are organizing these?

Reply:
Feel free to add that back.

Reply:
I went ahead and added it back in this pull request.